tests/ExchangeRateTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Stripe;
+
+class ExchangeRateTest extends TestCase
+{
+    public function testRetrieve()
+    {
+        $this->mockRequest(
+            'GET',
+            '/v1/exchange_rates/usd',
+            array(),
+            array(
+                'id' => 'usd',
+                'object' => 'exchange_rate',
+                'rates' => array('eur' => 0.845876),
+            )
+        );
+
+        $currency = "usd";
+        $rates = ExchangeRate::retrieve($currency);
+        $this->assertEquals('exchange_rate', $rates->object);
```

Comment:
I really wish we could do `$this->assertInstanceOf(ExchangeRate::class, $rates);` here instead, but unfortunately that doesn't fly with PHP <= 5.4 😢

Reply:
Ugh ...

Well, it's worth noting that [5.4 has been EOLed for more than two years](http://php.net/supported-versions.php) (since mid-2015) and hasn't even been receiving security fixes for more than a year, which means that it's not safe to use. This approach works fine for now, but it's totally plausible to drop support for it at this point. (If you're not updating off of PHP 5.4, you're also probably not bothering to update your version of stripe-php.)

Reply:
Actually, I misread that. It's _5.5_ has been EOLed for two years and hasn't been receiving updates for a year.

5.4 has been EOLed for three years (mid-2014) and hasn't been receiving updates for two years!

Reply:
http://php.net/supported-versions.php